inputfiles/addedTypes.jsonc

```diff
@@ -352,6 +352,95 @@
                     // https://developer.mozilla.org/en-US/docs/Web/API/WebXR_Device_API#browser_compatibility
                     "xr-spatial-tracking"
                 ]
+            },
+            "AutoFillBase": {
+                "name": "AutoFillBase",
+                "value": [
+                    // Off
+                    "off",
+                    // Automatic
+                    "on"
+                ]
+            },
+            "AutoFillAddressKind": {
+                "name": "AutoFillAddressKind",
+                "value": [
+                    "shipping",
+                    "billing"
+                ]
+            },
+            "AutoFillNormalField": {
+                "name": "AutoFillNormalField",
+                "value": [
+                    "name",
+                    "honorific-prefix",
+                    "given-name",
+                    "additional-name",
+                    "family-name",
+                    "honorific-suffix",
+
+                    "username",
+                    "new-password",
+                    "current-password",
+                    "one-time-code",
```

Comment:
A comment about iOS or mentioning Apple doc would be helpful to understand why this is here although WPT says no.